VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -321,5 +331,59 @@ public async Task<IActionResult> GetCurrentParticipantAsync(Guid conferenceId)
                 return StatusCode(e.StatusCode, e.Response);
             }
         }
+
+        [HttpPost("{conferenceId}/joinConference")]
+        [SwaggerOperation(OperationId = "StaffMemberJoinConference")]
+        [ProducesResponseType(typeof(ConferenceDetailsResponse), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [Authorize(AppRoles.StaffMember)]
+        public async Task<IActionResult> StaffMemberJoinConferenceAsync(Guid conferenceId,
+            StaffMemberJoinConferenceRequest request)
+        {
+            var username = request.Username.ToLower().Trim();
+            
+            try
+            {
+                var originalConference = await _videoApiClient.GetConferenceDetailsByIdAsync(conferenceId);
+
+                if (!_participantService.CanStaffMemberJoinConference(originalConference))
```

Comment:
we shouldn't need this check as we won't be calling this video web endpoint if the staff member is already not added in the conference